src/renderer/components/+catalog/catalog.tsx

```diff
@@ -214,23 +212,7 @@ export class Catalog extends React.Component<Props> {
     const isItemInHotbar = HotbarStore.getInstance().isAddedToActive(item.entity);
 
     return (
-      <div className={styles.entityName}>
-        {item.name}
-        <Icon
-          small
-          className={styles.pinIcon}
-          material={!isItemInHotbar && "push_pin"}
-          svg={isItemInHotbar && "push_off"}
-          tooltip={isItemInHotbar ? "Remove from Hotbar" : "Add to Hotbar"}
-          onClick={prevDefault(() => isItemInHotbar ? this.removeFromHotbar(item) : this.addToHotbar(item))}
-        />
-      </div>
-    );
-  }
-
-  renderIcon(item: CatalogEntityItem<CatalogEntity>) {
-    return (
-      <RenderDelay>
```

Comment:
I don't think you should remove the `<RenderDelay>`, it was added so that the catalogue would show up faster.

Reply:
I haven't noticed big difference and in general having `RenderDelay` for plain icon looks weird / ugly workaround.. 

_Solution:_ refactor `HotbarIcon` / figure out why it's slow? Most probably something wrong with material-ui thingy used inside..

Reply:
@aleksfront wdyt? I suggest to merge this PR and optimize `HotbarIcon` in separated request, maybe you can take it over if you'd like.

Reply:
I agree both. Removing `RenderDelay` is irrelevant to current PR, but `HotbarIcon` with material component should be refactored which is on todo. Let's proceed without `RenderDelay`.